### DISCUSSION TITLE
Index Page for Data Importer

### DISCUSSION THREAD

**Discussion by acatenac** [firefly-iii/firefly-iii#7249] [Firefly III]
The Data Importer appeared to install properly but when accessing from the public subdirectory I get a 404 error on the following redirect

http://dataimporter.caten.local/oauth/authorize?client_id=1&redirect_uri=http%3A%2F%2Fdataimport.caten.local%2Fcallback&response_type=code&scope=&state=2JXfdgGWknHVtxT842mqHoZ0usnpo6Kl1tXIlNta&code_challenge=VQOupiyFmJvRUGoPqVqhV2BQe3ZJJHE82vZ9S-zQ6T0&code_challenge_method=S256

Is there additional instructions for Data Importer setup  ... I followed the online docs where it seems like it should be relatively simple

Thanks

**Comment by JC5** [comment]
This is a link to the data importer, but it should be a link to Firefly III. Are you using a link to Firefly III for `FIREFLY_III_URL`?

**Comment by acatenac** [comment]
Nope :(   but now I am

|  |  |
| error | "invalid_client" |
| error_description | "Client authentication failed" |
| message | "Client authentication failed" |



Will see where this leads me
Very excited to use this update
Anthony Catenacci 


 

    On Sunday, March 19, 2023 at 02:30:17 AM EDT, James Cole ***@***.***> wrote:  
 
 


This is a link to the data importer, but it should be a link to Firefly III. Are you using a link to Firefly III for FIREFLY_III_URL?
